server/auth.go

```diff
@@ -525,6 +525,7 @@ func (s *Server) processClientOrLeafAuthentication(c *client, opts *Options) boo
 			c.Debugf("Account JWT not signed by trusted operator")
 			return false
 		}
+		// this only executes IF there's an issuer on the Juc - otherwise the account is already
```

Comment:
already what?

I know what, but seems like smth. is missing in this sentence

Reply:
Fixed.